test/sql/join/iejoin/test_iejoin_events.test

```diff
@@ -10,6 +10,9 @@ require vector_size 1024
 statement ok
 PRAGMA enable_verification
 
+statement ok
```

Comment:
Does this test get run at all with `require vector_size 1024`?

Reply:
Yes, that's a `>=` comparison:

```cpp
		auto required_vector_size = NumericCast<idx_t>(std::stoi(params[1]));
		if (STANDARD_VECTOR_SIZE < required_vector_size) {
			// vector size is too low for this test: skip it
			return RequireResult::MISSING;
		}
```

(agreed it's a bit confusing)